modules/nested-tabs/widgets/nested-tabs.php

```diff
@@ -184,10 +184,10 @@ protected function register_controls() {
 			],
 			'separator' => 'before',
 			'selectors_dictionary' => [
-				'top' => '--tabs-v2-direction: column; --tabs-v2-tabs-wrapper-direction: row; --tabs-v2-tabs-wrapper-width: initial;',
-				'bottom' => '--tabs-v2-direction: column-reverse; --tabs-v2-tabs-wrapper-direction: row; --tabs-v2-tabs-wrapper-width: initial;',
-				'end' => '--tabs-v2-direction: row-reverse; --tabs-v2-tabs-wrapper-direction: column; --tabs-v2-tabs-wrapper-width: 240px;',
-				'start' => '--tabs-v2-direction: row; --tabs-v2-tabs-wrapper-direction: column; --tabs-v2-tabs-wrapper-width: 240px;',
+				'top' => '--n-tabs-direction: column; --n-tabs-tabs-heading-direction: row; --n-tabs-tabs-heading-width: initial;',
+				'bottom' => '--n-tabs-direction: column-reverse; --n-tabs-tabs-heading-direction: row; --n-tabs-tabs-heading-width: initial;',
+				'end' => '--n-tabs-direction: row-reverse; --n-tabs-tabs-heading-direction: column; --n-tabs-tabs-heading-width: 240px;',
+				'start' => '--n-tabs-direction: row; --n-tabs-tabs-heading-direction: column; --n-tabs-tabs-heading-width: 240px;',
```

Comment:
I know this was the original variable name prefix, but `--n-tabs-tabs` looks a bit funny to me.

It looks like it is meant to target the tabs themselves inside the Nested Tabs widget, but it still looks a bit weird.
Maybe we should use just `--n-tab` for all of those instead? I'm not sure, what do you think @hein-obox ?

Reply:
@udidol Agree. All updated.